app/api/auth/auth.ts

```diff
@@ -1,19 +1,10 @@
 import GoogleProvider from 'next-auth/providers/google';
 
-import { env } from '@/env/server';
 export const authOptions = {
-  callbacks: {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    async signIn({ profile }: any) {
-      // TODO: Override default error message if returning false
-      const isAllowedToSignIn = profile.hd == 'nitkkr.ac.in';
-      return isAllowedToSignIn;
-    },
-  },
   providers: [
     GoogleProvider({
-      clientId: env.GOOGLE_CLIENT_ID!,
-      clientSecret: env.GOOGLE_CLIENT_SECRET!,
+      clientId: process.env.GOOGLE_CLIENT_ID!,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET!,
```

Comment:
what is error message

Reply:
The env import was removed, so there is no `env` namespace anymore. It's the default `process.env` in JS

Reply:
if process.env.GOOGLE_CLIENT_ID doesn't exist, we should fail with a good error message

Reply:
Yeah, the `!` is wrong, we should error out properly

Reply:
could add a check before accessing and if not found, throw error